cli/src/torii/tests.rs

```diff
@@ -213,7 +213,7 @@ struct QueryResponseTest {
 
 #[allow(variant_size_differences)]
 enum QueryResponseBody {
-    Ok(VersionedQueryResult),
+    Ok(Box<VersionedQueryResult>),
```

Comment:
After rebase clippy was warning me about very different variant sizes, so I have to Box it

Reply:
consider removing allow.